src/app/shared/theme.service.ts

```diff
@@ -232,6 +268,86 @@ export class ThemeService {
     this.applyMaterialPaletteToCss('warn', theme.material.colors.warn);
     this.applyTypographyToCss(theme.material.typography);
     this.applyAppColorsToCss(theme);
+    //this.getImageUrls(theme);
+  }
+
+  /*
+  getImageUrls(theme: Theme) {
+    theme.
+  }*/
+
+  /*
+  getThemeImage(apiPropertyKey: string): string | undefined {
+    let themeImage = themeImages.find((themeImage) => {
+      themeImage.apiPropertyKey === apiPropertyKey;
+    });
+
+    const url: string | undefined = themeImage
+      ? `${environment.apiEndpoint}/admin/properties/${themeImage.apiPropertyId}/image`
+      : undefined;
+    return url;
+  }*/
+
+  /*
+      return this.endpoints.apiProperties
+      .listPublic()
+      .pipe(map((response: ApiPropertyIndexResponse) => response.body.items));
+  */
+  /*
+  loadThemeImageUrls(): Observable<ThemeImage[]> {
+    return this.endpoints.apiProperties.listPublic().pipe(
+      switchMap((response: ApiPropertyIndexResponse) => {
+        return response.body.items.filter((prop) =>
+          prop.key.startsWith('explorer.theme.image')
+        );
+      }),
+      switchMap((apiProperties: ApiProperty[]) => {
+        return apiProperties.map((apiProperty) => {
+          return {
+            apiPropertyKey: apiProperty.key,
+            imageUrl:
+              apiProperty.value && this.isUUID(apiProperty.value)
+                ? `${environment.apiEndpoint}/admin/properties/${apiProperty.id}/image`
+                : undefined,
+          } as ThemeImage;
+        });
+      }),
+      catchError(() => {
+        this.toastr.error(
+          'There was a problem getting the configuration properties for images.'
+        );
+        return EMPTY;
+      })
+    );
+    / *
+      .subscribe((response: ApiPropertyIndexResponse) => {
+        const featureFlags = response.body.items.filter((prop) =>
+          prop.key.startsWith('explorer.theme.image')
+        );
+
+        themeImages = featureFlags.map((apiProperty) => {
+          return {
+            apiPropertyKey: apiProperty.key,
+            imageUrl: (apiProperty.value && isUUID(apiProperty.value))
+            ? `${environment.apiEndpoint}/admin/properties/${themeImage.apiPropertyId}/image`
+            : undefined;
+          } as ThemeImage;
+        });
+      });* /
+  }*/
+
+  private isUUID(value: string): boolean {
```

Comment:
This feels like a utility function to me. Perhaps it could be moved as a function to `shared.types.ts` or somewhere similar?

Reply:
Moved this function